csm_web/scheduler/views.py

```diff
@@ -110,6 +112,25 @@ def sections(self, request, pk=None):
         sections_by_day = self.get_sections_by_day(course)
         return Response({'userIsCoordinator': course.coordinator_set.filter(user=request.user).exists(), 'sections': sections_by_day})
 
+    # get a list of student information (for a selection of courses) to add to coord interface -- currently only used for download
+    @action(detail=False)
+    def students(self, request):
+        id_str = self.request.query_params.get('ids')
+        if not id_str or id_str == "/":
+            return Response({"students": None})
+        if id_str[-1] == "/":
+            id_str = id_str[:-1]
+        ids = id_str.split(",")
+        studs = Student.objects.select_related("user").filter(active=True, section__course__in=ids)
+
+        response = HttpResponse(content_type='text/csv')
+        response['Content-Disposition'] = 'attachment; filename="somefilename.csv"'
```

Comment:
Make the filename something more meaningful (e.g. "csm-cs61a-student-emails.csv")

Reply:
I don't want to include course names in the filename cause if the coord selects a large set of courses the name will be annoying.

Reply:
Perhaps just "scheduler-emails-export.csv" or something course-agnostic, but still meaningful.

Reply:
yeah I changed it to "csm-student-emails.csv"